doc/better-whitespace.txt

```diff
@@ -9,61 +9,107 @@ modes as well (see options below).
 To blacklist certain filetypes, set 'g:better_whitespace_filetypes_blacklist'
 to a list of the desired filetypes.
 
+COMMANDS                                        *better-whitespace-commands*
 
-                                                *ToggleWhitespace*
+:ToggleWhitespace                               *better-whitespace-:ToggleWhitespace*
+:EnableWhitespace                               *better-whitespace-:EnableWhitespace*
+:DisableWhitespace                              *better-whitespace-:DisableWhitespace*
 
-Call :ToggleWhitespace to toggle whitespace highlighting on/off.
+Call these functions to toggle whitespace highlighting on/off.
+This will set the highlighting of extraneous whitespace for the entire file.
 
-                                                *EnableWhitespace*
-
-To enable whitespace highlighting, call :EnableWhitespace. This will enable
+To disable whitespace highlighting, call :DisableWhitespace. This will disable
 highlighting of extraneous whitespace for the entire file.
 
-                                                *DisableWhitespace*
+:[range]NextTrailingWhitespace                   *better-whitespace-:NextTrailingWhitespace*
+:[range]PrevTrailingWhitespace                   *better-whitespace-:PrevTrailingWhitespace*
 
-To disable whitespace highlighting, call :DisableWhitespace. This will disable
-highlighting of extraneous whitespace for the entire file.
+These functions will respectively take you to the next and previous trailing whitespace
+in the ranged that they are passed (by default the whole file).
 
-                                                *StripWhitespace*
+:[range]StripWhitespace                         *better-whitespace-:StripWhitespace*
```

Comment:
I think the presence of the bang here is an error. The bang is only used for `:w!` as far as I know, so it can be removed from `:StripWhitespace`.

Or we should define it to say that it overrides `readonly` for example, then put it back and document it.

Reply:
Went with the latter option.